ping/rust/src/bin/testplan_0500.rs

```diff
@@ -12,7 +12,7 @@ async fn main() -> Result<()> {
     let local_key = identity::Keypair::generate_ed25519();
     let local_peer_id = PeerId::from(local_key.public());
 
-    let swarm = OrphanRuleWorkaround(Swarm::new(
+    let swarm = OrphanRuleWorkaround(Swarm::with_async_std_executor(
```

Comment:
```
   Compiling testplan v0.1.0 (/usr/src/testplan/plan)
error[E0599]: no function or associated item named `with_async_std_executor` found for struct `libp2p::Swarm` in the current scope
  --> src/bin/testplan_0500.rs:15:45
   |
15 |     let swarm = OrphanRuleWorkaround(Swarm::with_async_std_executor(
   |                                             ^^^^^^^^^^^^^^^^^^^^^^^ function or associated item not found in `libp2p::Swarm<_>`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `testplan` due to previous error
```
```suggestion
    let swarm = OrphanRuleWorkaround(Swarm::with_tokio_executor(
```

We are using tokio as an executor here. `libp2p` is configured with the `tokio` feature and not the `async-io` feature.